### PR TITLE
document for the new aws market-place

### PR DESCRIPTION
**Pull request by punitmundra**
## Description

[Please describe what this change achieves]
This PR for the AWS market place 
Product : Automate
## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/aws_marketplace.md [hunk 1]
```diff
@@ -16,557 +16,114 @@ product = ["client", "workstation"]
...
-    unzip starter_kit.zip
-    cd starter_kit/chef-repo
-    ```
+In case you have a need of running automate in HA mode, please contact [Chef Support](https://www.chef.io/support).
 
-2.  {{% install_aws_chef_server_knife_client_list %}}
+## Installation
 
-## Configure backups
+TODO: Select the Chef Automate AWS Marketplace [CloudFormation template](NEED LINK
```
Comment (punitmundra) [inline content/aws_marketplace.md]: https://aws-ami-chef-automate-v2.s3.amazonaws.com/cloudformation_template.yaml

Comment (punitmundra) [reply content/aws_marketplace.md]: @kagarmoe : I have added the link. let me know. if any thing is needed apart from that

File: content/aws_marketplace.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ draft = false
 
 gh_repo = "chef-web-docs"
 
-aliases = ["/aws_marketplace.html"]
+aliases = ["/aws_marketplace.html", "/aws_ami.html"]
```
Comment (IanMadd) [inline content/aws_marketplace.md:7]: We don't need this change, there's no current aws_ami page.

File: content/aws_marketplace.md [hunk 3]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
-    CloudFormation stack will have the setup URL in the `Outputs`
-    section.
-
-    {{< note spaces=4 >}}
-
-    {{% notes_chef_aws_ssl %}}
+Chef Automate is an enterprise platform that allows developers, operations, and security engineers to collaborate on application and infrastructure changes with speed and at scale. Chef Automate provides actionable insights across data centers and cloud providers, wherever your nodes live.
 
-    {{< /note >}}
+Chef Automate is the center of the modern Chef platform, providing users with a single source of truth for infrastructure, security, and application automation. The comprehensive dashboard provides real-time views of your configuration management activity. Chef Automate comes bundled with the latest Chef Infra Server, providing the core tools you need to manage your enterprise infrastructure. Data collection is enabled by default, allowing your nodes to report activity in real-time. This instance is free for 60 days or you can bring your license (BYOL).
```
Comment (IanMadd) [inline content/aws_marketplace.md]:
```suggestion
Chef Automate is the center of the modern Chef platform, providing users with a single source of truth for infrastructure, security, and application automation. The comprehensive dashboard provides real-time views of your configuration management activity. Chef Automate comes bundled with the latest Chef Infra Server, providing the core tools you need to manage your enterprise infrastructure. Data collection is enabled by default, allowing your nodes to report activity in real-time. This instance is free for 60 days or you can bring your own license (BYOL).
```

File: content/aws_marketplace.md [hunk 4]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
-    ID of your Chef Automate instance. The instance ID is required for
-    authorization to access the setup page.
-
-3.  Fill out the setup form and submit it.
+Use this instance with Chef Workstation installed on your laptop or a separate AWS instance.
 
-4.  Follow the link and log into Chef Automate.
-
-## Configure the workstation
+{{< workstation >}}
```
Comment (IanMadd) [inline content/aws_marketplace.md]: {{< shortcode_name >}} is for HTML shortcodes 
{{% shortcode_name %}} is for MD shortcodes, we've already got this shortcode from #3203 and it would be better to keep it in markdown.

```suggestion
{{% workstation %}}
```

File: content/aws_marketplace.md [hunk 5]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
+1. Give Chef Automate an additional five minutes for all the services to start running.
 
-1.  Run `sudo -i` to gain administrator privileges.
+![Select next to create stack](/images/StackDetails.png "Stack Details")
 
-2.  Run `chef-marketplace-ctl hostname` to view the current hostname.
+## Post-Installation
 
-3.  Configure the `api_fqdn` in `/etc/chef-marketplace/marketplace.rb`
+1. Navigate to the AWS deployment **output** tab and locate the Chef Automate URL, user name, and password. You will need these in the next steps.
```
Comment (IanMadd) [inline content/aws_marketplace.md]: The list items in this section are formatted as ordered and unordered list items, it would be better to go with ordered list items for all of them.

File: content/aws_marketplace.md [hunk 6]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
+- Congratulations! You've started Chef Automate!
 
-To edit the Amazon Machine Images (AMI) instance size, do the following:
+![ ](/images/DashboardsPage.png "Dashboards Page")
 
-1.  Login using SSH to access the Chef Automate instance. Use the SSH
-    key pair and the IP address or public hostname that was
-    automatically assigned when the Amazon Machine Images (AMI) was
-    launched. The default user is `ec2-user`. For example:
+## Add Chef Servers
```
Comment (IanMadd) [inline content/aws_marketplace.md:80]: The ordered list item numbers aren't being formatted correctly by Hugo. The empty line between the starting list item paragraph and the additional text/images of that list item seems to be throwing Hugo off. 

So this isn't working:

```
1. Add Chef-Server Details, select the Add Chef Infra Server Button.

  ![ ](/images/AddChefServer.png "Add Chef Server")
  
1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.  
```


But this does:

```
1. Add Chef-Server Details, select the Add Chef Infra Server Button.
  ![ ](/images/AddChefServer.png "Add Chef Server")
  
1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.  
```

File: content/aws_marketplace.md [hunk 7]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
+    - Admin Key: _copy the key from starter kit_
 
-10. Use SSH to log into the new instance. Use the SSH key pair and new
-    IP address:
+1. Select **Add Chef Organization**.
 
-    ```bash
-    ssh -i /path/to/ssh_key.pem ec2-user@<instance IP address>
-    ```
+  <img src="/images/OrgPageDetails.png" style="width: 30%" />
```
Comment (IanMadd) [inline content/aws_marketplace.md]: Might change this to the figure shortcode. Also, we've already got this image (and some others) from the automate repo, so we might as well reuse those images instead of adding new ones.

https://docs.chef.io/automate/chef_infra_server/

```suggestion
  {{< figure src="/images/automate/add-chef-organization-popup-menu.png" width="500" alt="Add Chef Organization Form">}}
```

File: content/aws_marketplace.md [hunk 8]
```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
...
 
-11. Follow the [instructions for changing the
-    hostname](#change-automate-hostname)
+## AWS Deployment Security
```
Comment (IanMadd) [inline content/aws_marketplace.md:108]: Same problem with list item numbers as above in the **Add Chef Servers** section

File: themes/docs-new/layouts/shortcodes/workstation.md [hunk 9]
```diff
@@ -1,8 +1,10 @@
-Chef Workstation includes:
+<p>Chef Workstation includes:</p>
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/workstation.md]: Undo the changes to this file. It should be in MD.

**Review by IanMadd — COMMENTED** [review]
Thoughts...

File: content/aws_marketplace.md (hunk 2 above)
Comment (kagarmoe) [reply content/aws_marketplace.md:7]: They may have used it in marketing collateral. Would this work if they have something that links to that docs page (which doesn't exist)?

File: themes/docs-new/layouts/shortcodes/workstation.md (hunk 9 above)
Comment (kagarmoe) [reply themes/docs-new/layouts/shortcodes/workstation.md]: List doesn't render in markdown

File: content/aws_marketplace.md (hunk 2 above)
Comment (IanMadd) [reply content/aws_marketplace.md:7]: I just checked, there was never an aws_ami page. This would work, but it would be better if links were fixed. Was there a page with that name in another section like automate or server?

File: content/aws_marketplace.md [hunk 10]
```diff
@@ -4,7 +4,7 @@ draft = false
 
 gh_repo = "chef-web-docs"
 
-aliases = ["/aws_marketplace.html"]
+aliases = ["/aws_marketplace.html", "/aws_ami.html"]
 
 product = ["client", "workstation"]
```
Comment (IanMadd) [inline content/aws_marketplace.md]: Products should be updated for search. I think this is right:

```suggestion
product = ["server", "workstation", "automate"]
```

File: themes/docs-new/layouts/shortcodes/workstation.md (hunk 9 above)
Comment (IanMadd) [reply themes/docs-new/layouts/shortcodes/workstation.md]: I just tested it, it works fine for me.

File: content/aws_marketplace.md (hunk 2 above)
Comment (kagarmoe) [reply content/aws_marketplace.md:7]: I don't know, but we can fix it next week in a different PR.

File: content/aws_marketplace.md (hunk 10 above)
Comment (kagarmoe) [reply content/aws_marketplace.md]: Excellent catch.

**Review by IanMadd — APPROVED** [review]
